File: src/test/java/chapter/eleven/two/one/SetExampleTest.java
package chapter.eleven.two.one;

import org.junit.jupiter.api.Test;

import static chapter.eleven.two.one.SetExample.main;

class SetExampleTest {

    @Test
    void mainTest() {

        main(new String[]{"hello", "world"});
        main(new String[]{"she", "said", "she", "is", "signorina"});
    }
}

File: src/test/kotlin/chapter/eleven/two/one/SetExampleTest.kt
package chapter.eleven.two.one

import org.junit.jupiter.api.Test

internal class SetExampleTest {
    @Test
    fun mainTest() {
        SetExample.main(arrayOf("hello", "world"))
    }

    @Test
    fun mainTest2() {
        SetExample.main(arrayOf("she", "said", "she", "is", "signorina"))
    }
}